grader/reject.go

```diff
@@ -125,3 +135,28 @@ func checkSubmissionLimitWindow(window *model.SubmittionLimitWindow,
 
     return nil, nil;
 }
+
+func checkLateAcknowledgment(assignment *model.Assignment, lateAcknowledgment bool) (RejectReason, error) {
+    // if the user acknowledges that they are submitting late, do not reject the submission
+    if lateAcknowledgment {
+        return nil, nil
+    }
+
+    policy := assignment.GetLatePolicy()
+    
+    if policy.Type == model.EmptyPolicy || policy.Type == model.BaselinePolicy {
```

Comment:
Why is baseline policy skipped?
Since that one is checked for (full) rejection, seems like it should be checked here as well.

Reply:
Take a closer look at late policies.
I believe that the baseline policy does not reject if the submission is just late, only if it past the rejection date.